saleor/graphql/order/utils.py

```diff
@@ -300,9 +300,12 @@ def prepare_insufficient_stock_order_validation_errors(exc):
             if item.warehouse_pk
             else None
         )
+
+        item_name = item.order_line or item.variant
+
         errors.append(
             ValidationError(
-                f"Insufficient product stock: {item.order_line or item.variant}",
+                f"Insufficient product stock: {str(item_name).replace('%', '%%')}",
```

Comment:
We could entirely drop rendering the product name or ID from the message since there is a dedicated field in error for that - `order_lines` - where we return IDs of items that throw this error, right?

Reply:
yes, so I think this will be best idea